build.gradle

```diff
@@ -74,21 +74,24 @@ configurations {
 }
 
 dependencies {
-    compile 'com.datastax.cassandra:cassandra-driver-core:2.1.8'
+    compile 'com.datastax.cassandra:cassandra-driver-core:3.1.0'
     compile 'org.slf4j:slf4j-api:1.7.12'
     compile 'com.google.code.findbugs:jsr305:3.0.0'
 
     testCompile 'org.apache.logging.log4j:log4j-core:2.2'
     testCompile 'junit:junit:4.12'
     testCompile 'org.assertj:assertj-core:2.0.0'
+    testCompile 'org.assertj:assertj-joda-time:2.0.0'
+    testCompile 'joda-time:joda-time:2.9.4'
```

Comment:
I think we'd prefer not to use `joda-time`. I can see you've used it in the `CqlMigratorImplTest` so I'll leave a comment there on how to write those tests without it.